Create 11-23-2021-scyrill-project-update.md

scyrill opened the pull request.

scyrill committed:
Create 11-23-2021-scyrill-project-update.md

scyrill committed:
Create 11-23-2021-scyrill-project-update.md

MishaCienfuegos commented on _posts/scyrill/11-23-2021-scyrill-project-update.md:
````diff
@@ -0,0 +1,158 @@
+```python 
+for line in pt_notes: 
+  #figure out why this isn't printing 'final diagnosis'
+  re.findall(‘.+\s Diagnos.s’):
````
The fact that this line is highlighted in wine red makes me think something is amiss here in your post and possibly in your Trinket code. These angled single quotes `‘` are separate characters from straight single quotes `'`. Same is true for angled `“` vs. straight `"` double quotes. Could you replace them and see how things look/run?

scyrill replied:
Sure, thanks for checking this for me, I noticed this, but wasn't sure if it would cause a problem or not.

MishaCienfuegos commented on _posts/scyrill/11-23-2021-scyrill-project-update.md:
````diff
@@ -0,0 +1,158 @@
+bank.append(dc[0])
+print(bank)
+```
+This finally worked! It returned all the diagnoses, but it included the whitespace character ```\n```, which was something I wanted to get rid of. I knew that I could use ```strip``` to take out the whitespace, so I tried using ```rstrip()``` to do so. I chose to do this by introducing another variable called ```entries```, attaching ```rstrip()```, and putting into what I wanted it to strip out  ...
````
Again, the wine red highlighting makes me think something is getting mixed up here. Try just one backtick before and after the `\n` (and the other in-line code highlighted) and see if that clears things up.

In markdown you can use one backtick for in-line code and three for multiline code blocks.

scyrill replied:
Got it, but this makes me wonder why this happened here and not other points in the code, like when I referred to regex or other areas?

scyrill replied:
I added new commit addressing this issue, but things are still showing up in this wine red highlight, not sure why though @MishaCienfuegos

MishaCienfuegos replied:
I think you're right, that this is actually being caused by things happening elsewhere in this file. What happens if you replace the angle single `‘’` and double quotes `“”` with straight ones? As you notice, when you're doing things like writing regex expressions within code blocks they may be causing problems.

scyrill replied:
I straightened them out but the same issue is happening in the same place

MishaCienfuegos replied:
Looks to me like your most recent changes fix things. Now when I look at the [files changed](https://github.com/inf380p/inf380p.github.io/pull/563/files) on this pull request the wine red syntax highlighting is gone. Does that look right to you?

Thanks for working through this with me!

MishaCienfuegos reviewed (changes requested):
This update is looking good, lots of reflection and code examples! I left a few comments about markdown and character considerations that may help your post show up better. Let me know if you have questions.

MishaCienfuegos commented on _posts/scyrill/11-23-2021-scyrill-project-update.md:
```diff
@@ -0,0 +1,158 @@
+---
```
In the file title above, you may want to change the file name to match the date structure in [your other post files](https://github.com/inf380p/inf380p.github.io/tree/production/_posts/scyrill). The date syntax (`YYYY-MM-DD-`) is important to the code that builds the website, so your post may not show up where you think it should if it isn't written exactly like it expects.

scyrill committed:
Update and rename 11-23-2021-scyrill-project-update.md to 2021-11-23-scyrill-project-update.md

Replaced ``` with `

scyrill committed:
Update 2021-11-23-scyrill-project-update.md

Went through and changed this double quotes to single ones, but some areas are still highlighting in the wine red color. Not sure why this is

scyrill committed:
Update 2021-11-23-scyrill-project-update.md

MishaCienfuegos reviewed (approved):
Thanks for hunting down the issue with the quotes with me! This looks good.
